### PR TITLE
Bank transfer: Hide email form if email was already sent (Z#23128941)

### PR DESCRIPTION
**Pull request by raphaelm**

### CODE REVIEW

**Review by pc-coholic — APPROVED** [review]

File: src/pretix/plugins/banktransfer/templates/pretixplugins/banktransfer/pending.html [hunk 1]
```diff
@@ -106,7 +106,20 @@
 {% endif %}
 
 {% if invoice_email_enabled and has_invoices %}
-    <form method="post" action="{% eventurl event "plugins:banktransfer:mail_invoice" order=order.code secret=order.secret %}">
+    {% if info.send_invoice_to %}
+        <p>
+            {% blocktrans trimmed with recipient=info.send_invoice_to %}
+                At your request, we are sending the invoice directly to {{ recipient }}.
```
Comment (luelista) [inline src/pretix/plugins/banktransfer/templates/pretixplugins/banktransfer/pending.html:112]: Usually when this message is displayed, the invoice was already sent (only exception I can think of is if it's still in the task queue for actually sending the email), right? So we should better use past tense here

Comment (luelista) [inline src/pretix/plugins/banktransfer/templates/pretixplugins/banktransfer/pending.html:112]:
```suggestion
                At your request, we sent the invoice directly to {{ recipient }}.
```

File: src/pretix/plugins/banktransfer/payment.py [hunk 2]
```diff
@@ -530,6 +530,7 @@ def payment_pending_render(self, request: HttpRequest, payment: OrderPayment):
             'code': self._code(payment.order),
             'order': payment.order,
             'amount': payment.amount,
+            'info': payment.info_data,
```
Comment (luelista) [inline src/pretix/plugins/banktransfer/payment.py:533]: I would recommend a somewhat less ambiguous variable name than `info`. Either `payment_info`, `info_data` or `payment_info_data`.
```suggestion
            'payment_info': payment.info_data,
```

**Review by luelista — APPROVED** [review]
Two remarks, apart from that LGTM :)

File: src/pretix/plugins/banktransfer/templates/pretixplugins/banktransfer/pending.html (hunk 1 above)
Comment (raphaelm) [reply src/pretix/plugins/banktransfer/templates/pretixplugins/banktransfer/pending.html:112]: Yes. I was worried about the task not yet executed or the email not sent successfully, but let's not bother the custoemr with it.

**Pull request merged**